packages/design-system-twrnc-preset/src/Theme/Theme.providers.tsx

```diff
@@ -23,45 +16,18 @@ export const ThemeContext = createContext<ThemeContextProps>(
 
 export const ThemeProvider: React.FC<ThemeProviderProps> = ({
   children,
-  colorSet = ColorSet.Brand,
-  theme = Theme.Default,
+  theme,
 }) => {
-  const [currentColorSet, setCurrentColorSet] = useState<ColorSet>(colorSet);
-  const [currentTheme, setCurrentTheme] = useState<Theme>(theme);
-  const systemColorScheme = useColorScheme(); // 'light' | 'dark' | null
-
-  const activeColorScheme: ColorScheme = useMemo(() => {
-    if (currentTheme === Theme.Default) {
-      return systemColorScheme === 'dark'
-        ? ColorScheme.Dark
-        : ColorScheme.Light;
-    }
-    if (currentTheme === Theme.Light) {
-      return ColorScheme.Light;
-    }
-    if (currentTheme === Theme.Dark) {
-      return ColorScheme.Dark;
-    }
-    throw new Error('Invalid theme value');
-  }, [currentTheme, systemColorScheme]);
-
   const tw = useMemo(() => {
-    const tailwindConfig = generateTailwindConfig(
-      currentColorSet,
-      activeColorScheme,
-    );
+    const tailwindConfig = generateTailwindConfig(theme);
     return create(tailwindConfig);
-  }, [currentColorSet, activeColorScheme]);
+  }, [theme]);
 
   return (
     <ThemeContext.Provider
       value={{
         tw,
-        colorSet: currentColorSet,
-        colorSetList: colorSetList[colorSet][activeColorScheme],
-        theme: currentTheme,
-        setColorSet: setCurrentColorSet,
-        setTheme: setCurrentTheme,
+        theme,
```

Comment:
Provider now only returns `tw` for the useTailwind hook and `theme` for `useTheme` hook